core/templates/app.constants.ts

```diff
@@ -161,7 +161,8 @@ export class AppConstants {
     TOPIC: 'topic',
     SKILL: 'skill',
     STORY: 'story',
-    QUESTION: 'question'
+    QUESTION: 'question',
+    SUGGESTION: 'suggestion'
```

Comment:
Can we add all of this in alphabetical order?

Reply:
done